includes/class-wc-stripe-order-helper.php

```
@@ -516,6 +523,34 @@ public function update_stripe_checkout_session_id( ?WC_Order $order = null, stri
 		return $this->update_order_meta( $order, self::META_STRIPE_CHECKOUT_SESSION_ID, $checkout_session_id );
 	}
 
+	/**
+	 * Gets whether the payment method should be saved to the store after a checkout session payment.
+	 *
+	 * @param WC_Order|null $order
+	 * @return bool
+	 */
+	public function get_should_save_stripe_payment_method( ?WC_Order $order = null ): bool {
+		return wc_string_to_bool( $this->get_order_meta( $order, self::META_STRIPE_SHOULD_SAVE_PAYMENT_METHOD ) );
+	}
```

PHPMD (2.15.0)

[error] The method get_should_save_stripe_payment_method is not named in camelCase. (undefined)

(CamelCaseMethodName)